DEPLOYMENT.md

```diff
@@ -65,13 +103,20 @@ IMAGE runs entirely through Docker Compose. You’ll see several Compose files i
 - test-docker-compose.yml — overrides for test/dev setups (for us, unicorn.cim.mcgill.ca). 
 - prod-docker-compose.yml — overrides for production setups (for us, pegasus.cim.mcgill.ca).
 - ec2-docker-compose.yml — overrides for our EC2 instance.
-- docker-compose.override.yml — local overrides for testing.
+- docker-compose.override.yml — optional overrides for local development.
 
 Docker Compose lets you layer files with -f flags, or list them in .env via COMPOSE_FILE. The next section covers how to populate the .env file.
 
 # Environment Configuration
 
-In the same directory, make the following soft links, to get access to scripts and configuration, and copy the [prod-docker-compose.yml](https://github.com/Shared-Reality-Lab/IMAGE-server/blob/main/prod-docker-compose.yml) file so you can modify it to reflect your server configuration:
+We place everything under `/var/docker/image` to keep a clean separation between:
+- The core IMAGE-server repository (`/var/docker/image/IMAGE-server`) — this is the Git repo you clone and keep updated.
+- The server’s working directory (`/var/docker/image`) — this is where you run commands, keep local configs, and apply overrides.
+
+Think of `/var/docker/image` as the control room for the server. The source code (IMAGE-server) lives in a room. We pull out just the key controls (scripts, configs, compose files) into the control room with symlinks, so operators only need to work there. When the source code updates, the control room “sees” the changes instantly, but its own local switches and dials (prod-docker-compose.yml, .env) stay safe.
```

Comment:
I see what you're attempting here, but I think we need a consistent tone that the reader is competent and doesn't need metaphors, just the important content. I think this paragraph can be safely deleted.